src/Storages/System/StorageSystemTables.cpp

```diff
@@ -239,6 +245,38 @@ class TablesBlockSource : public ISource
     String getName() const override { return "Tables"; }
 
 protected:
+    NameToNameMap getSelectParamters(const StorageMetadataPtr & metadata_snapshot)
+    {
+        const SelectQueryDescription & query_description = metadata_snapshot->getSelectQuery();
+        ASTPtr inner_query = query_description.inner_query;
+        if (!inner_query || !inner_query->as<ASTSelectWithUnionQuery>())
+            return {};
+
+        return inner_query->as<ASTSelectWithUnionQuery>()->getQueryParameters();
+    }
+
+    void FillParametralizedViewData(MutableColumns & columns, const StoragePtr & table, size_t & res_index, size_t & src_index)
+    {
+        if (src_index == 14 && columns_mask[src_index++])
+        {
+            if (table)
+            {
+                StorageMetadataPtr metadata_snapshot = table->getInMemoryMetadataPtr();
+
+                NameToNameMap query_params = getSelectParamters(metadata_snapshot);
+                if (!query_params.empty())
+                {
+                    Array changes;
```

Comment:
I think we should change this variable name as its not related to changes, maybe something like 'query_parameters_array'